package.json

```diff
@@ -1,42 +1,16 @@
 {
-    "name": "@oracle/netsuite-suitecloud-node-cli",
-    "version": "1.0.0",
-    "description": "SuiteCloud CLI for Node.js",
-    "keywords": [
-        "NetSuite",
-        "SuiteCloud",
-        "SDF"
-    ],
-    "bugs": {
-        "url": "https://github.com/oracle/netsuite-suitecloud-node-cli/issues"
-    },
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/oracle/netsuite-suitecloud-node-cli.git"
-    },
-    "license": "UPL-1.0",
-    "main": "src/suitecloud.js",
-    "preferGlobal": true,
-    "sdkDownloadUrl": "https://system.netsuite.com/download/suitecloud-sdk/cli/java/latest",
-    "bin": {
-        "suitecloud": "src/suitecloud.js"
-    },
-    "dependencies": {
-        "chalk": "^2.4.2",
-        "cli-spinner": "^0.2.10",
-        "commander": "^2.20.0",
-        "inquirer": "^6.3.1",
-        "request-promise-native": "^1.0.7",
-        "sdf-cli-local": "file:./src/sdf-cli-local",
-        "suitecloud-testing-jest-configuration": "file:./testing-framework",
-        "vm2": "^3.8.0",
-        "xml2js": "^0.4.19"
-    },
-    "devDependencies": {
-        "jest": "^24.7.1"
-    },
-    "scripts": {
-        "test": "jest",
-        "postinstall": "node postinstall.js"
-    }
+  "name": "@oracle/netsuite-suitecloud",
+  "version": "1.0.0",
+  "private": true,
+  "bin": {
+    "suitecloud": "./packages/node-cli/src/suitecloud.js"
```

Comment:
Is this necessary here?

Reply:
Yes, so anyone can do npm install && npm link from the root repository (in case that they downloaded the repository from git, and not installed @oracle/netsuite-suitecloud-nodejs-cli package from npm)

Reply:
I understand the use case, but still I think I would keep it simple and not have that option.
Customers are currently downloading the code and installing from the repo because we don't have the npm package setup. However, in the future we should not support this, only npm installations.